src/ess/loki/workflow.py

```diff
@@ -15,10 +14,10 @@ def live_workflow(group: JSONGroup) -> WorkflowResult:
     _ = group
     return {
         'IofQ': sc.DataArray(
-            data=sc.array(dims=['Q'], values=np.zeros(101)),
+            data=sc.zeros(dims=['Q'], shape=[100]),
             coords={
                 'Q': sc.linspace(
-                    dim='Q', start=0.01, stop=0.3, num=101, unit='1/angstrom'
+                    dim='Q', start=0.01, stop=0.3, num=100, unit='1/angstrom'
```

Comment:
Sorry, I was unclear: The data has length 100, but the coord length 101, i.e., this is a histogram where `Q` gives the bin edges.